internal/integration/apprise/apprise.go

```diff
@@ -38,8 +38,9 @@ func (c *Client) SendNotification(entry *model.Entry) error {
 	}
 
 	requestBody, err := json.Marshal(map[string]any{
-		"urls": c.servicesURL,
-		"body": message,
+		"urls":  c.servicesURL,
+		"body":  message,
+		"title": entry.Feed.Title,
```

Comment:
> I don't think `entry.Feed.Title` is populated in this case. It would be more appropriate to pass `feed *model.Feed` to this function and use `feed.Title` (similar to other integrations like Nfty or Telegram).

Thank you for your review. I have modified it according to the implementation of Ntfy. Please review it again.